syntax = "proto3";
import "google/protobuf/empty.proto";

message Point3 {
    float x=1;
    float y=2;
    float z=3;
}

service addToScene {
    rpc addPoint(Point3) returns (google.protobuf.Empty){}
}